### PR TITLE
Minor cleanup of CSSStyleSheet»insertRule() doc

### PR DESCRIPTION
**Pull request by sideshowbarker**
This is a follow-up to https://github.com/mdn/content/pull/12219.

### CODE REVIEW

**Comment by sideshowbarker** [comment]
@wbamberg ping?

File: files/en-us/web/api/cssstylesheet/insertrule/index.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ browser-compat: api.CSSStyleSheet.insertRule
 {{APIRef("CSSOM")}}
 
 The **`CSSStyleSheet.insertRule()`**
-method inserts a new [CSS rule](/en-US/docs/Web/API/CSSRule) into the [current style sheet](/en-US/docs/Web/API/CSSStyleSheet), with some [restrictions](#restrictions).
+method inserts a new [CSS rule](/en-US/docs/Web/API/CSSRule) into the [current style sheet](/en-US/docs/Web/API/CSSStyleSheet).
+
+CSS has some intuitive and not-so-intuitive restrictions affecting where rules can be inserted. There are a number of [exceptions](#exceptions) that will be raised when those rules are violated.
```
Comment (wbamberg) [inline files/en-us/web/api/cssstylesheet/insertrule/index.md:17]:
```suggestion
CSS has some restrictions affecting where rules can be inserted. There are a number of [exceptions](#exceptions) that will be raised when those rules are violated.
```

...although if it were me I'd just delete this paragraph. The API can throw exceptions: so can lots of APIs, but we don't generally call out this fact.

Comment (sideshowbarker) [reply files/en-us/web/api/cssstylesheet/insertrule/index.md:17]: > ...although if it were me I'd just delete this paragraph. The API can throw exceptions: so can lots of APIs, but we don't generally call out this fact.

Yeah, I felt the same way about it — so let’s just go ahead and drop it

Comment (sideshowbarker) [inline files/en-us/web/api/cssstylesheet/insertrule/index.md:17]:
```suggestion
```

**Review by wbamberg — APPROVED** [review]
👍 thanks @sideshowbarker !

**Pull request merged**